tests/rules/doc-string-matching/single-pattern.yaml

```diff
@@ -0,0 +1,12 @@
+rules:
+  - id: doc-string-comment
+    options:
+      match_on_doc_comments: true
```

Comment:
We need some test with this option disabled and a `/** ... */` pattern, and test that we match ignoring whatever the doc-comment pattern says.

Reply:
I've added a test for this now.